### PR TITLE
Fix save/create new uuid_and_cookie file.

### PR DESCRIPTION
**Pull request by Tkd-Alex**
Fix for #1021

### CODE REVIEW

File: instabot/api/api.py [hunk 1]
```diff
@@ -90,6 +90,8 @@ def set_user(self, username, password, generate_all_uuids=True, set_device=True)
         self.username = username
         self.password = password
 
+        self.logger = logging.getLogger("[instabot_{}]".format(self.username))
```
Comment (ohld) [inline instabot/api/api.py:93]: is it the best way to get access to logger? I know that Bot() class has its own logger too.

**Comment by ohld** [comment]
Are you sure that this is the best way to import logger? We have a lot of shitcode in all instabot and in my opinion your way of initialising the logger is another piece of shitcode ........ Maybe I am wrong. 

Moreover, We have logger in `Bot()` class too. Maybe we need to replace Bot's logger with API' that you just introduced too.

**Comment by ohld** [comment]
The easiest way to merge this PR - is to remove logger at all. We can make it pretty later.

This PR is important because it fixes one big issue. Big thanks for it. Let's remove logger at all from this PR and I'll merge with pleasure.

**Comment by Tkd-Alex** [comment]
The logger has always been like this. I've only replace the **id** with **username**. :confused: 
The logger in _bot.py_ it's only a reference of _api.py_
https://github.com/instagrambot/instabot/blob/master/instabot/bot/bot.py#L280

**Review by ohld — APPROVED** [review]

**Comment by ohld** [comment]
Sorry, thanks for the commits. Changes are already in pip

**Pull request merged**